integration-tests/gradle/src/test/resources/test-resources-in-build-steps/application/build.gradle

```diff
@@ -19,11 +19,11 @@ test {
     forkEvery 1
 }
 
-quarkusDev {
+quarkusPrepare {
```

Comment:
@glefloch could you explain to me what does change does?

Reply:
before we were specifying that `quarkusDev` must be run after the `publish` (or install) of the `runtime` and `deployment` module of the project. 
Since `quarkusPrepare` is run before `quarkusDev` but needs these dependencies to build the `AppModel`, I change this to make the `quarkusPrepare` depends on the publish of the required module (which are not always dependencies like for the `deployment`module)

Reply:
makes sense, thanks!